src/components/Modals/SlotMenu.vue

```diff
@@ -0,0 +1,123 @@
+<template>
+  <teleport-modal
+    content-class="content-slotmenu"
+    :isSimpleModal="true"
+    :modelValue="modelValue"
```

Comment:
More specifically, we don't need to pass down modelValue anymore, and just use v-if on the parent component and emit up to it. Could you remove this and follow the new style from #499 ?